src/modules/storage.js

```diff
@@ -0,0 +1,12 @@
+const saveStorage = (items) => {
+  localStorage.setItem('todo-list', JSON.stringify(items));
+};
+
+const getStorage = () => {
+  const tasks = localStorage.getItem('todo-list')
+    ? JSON.parse(localStorage.getItem('todo-list'))
+    : [];
+  return tasks;
+};
+
+export { saveStorage, getStorage };
```

Comment:
- [ ] Please let's implement the changes requested by the previous reviewer. Tasks added to the list are not still saved in the local storage.

Reply:
Done